pkg/label/label_selinux.go

```diff
@@ -41,7 +43,7 @@ func InitLabels(options []string) (string, string, error) {
 				return "", "", nil
 			}
 			if i := strings.Index(opt, ":"); i == -1 {
-				return "", "", fmt.Errorf("Bad SELinux Option")
+				return "", "", fmt.Errorf("bad SELinux Option")
```

Comment:
`errors.New`? And lowercase the "Option" word, please.

Bad SELinux Option. We Are New York Times. Titlecase Your Messages, Blah Blah Blah, Resistance Is Futile.